<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Allocator</class>
 <widget class="QMainWindow" name="Allocator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Allocator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="addHoleBtn">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>110</y>
      <width>87</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>First Fit</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Worst Fit</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Best Fit</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Memory of 1024 KB</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="holeBaseAddrTextEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>290</y>
      <width>87</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Algorithm</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Hole Size</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="processSizeTextEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addProcessBtn">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>200</y>
      <width>87</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="holeSizeTextEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Starting Address</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Process Size</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showBtn">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>200</y>
      <width>87</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Allocate</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>50</y>
      <width>141</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="showDiagram">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>290</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Diagram</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>350</y>
      <width>371</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Note: There is no validation, so please enter correct values !</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>591</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
